clar2wasm/tests/wasm-generation/noop.rs

```diff
@@ -0,0 +1,51 @@
+use clar2wasm::tools::crosscheck;
+use clarity::vm::Value;
+use proptest::arbitrary::any;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+use crate::PropValue;
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_noop_to_uint(
+        val in any::<i128>()
+        .prop_filter("non-negative", |v| v >= &0i128)
+    ) {
+        crosscheck(
+            &format!("(to-uint {})", PropValue(Value::Int(val))),
+            Ok(Some(Value::UInt(val.try_into().unwrap())))
+        )
+    }
+}
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_noop_to_uint_err(
+        val in any::<i128>()
+        .prop_filter("negative", |v| v < &0i128)
+    ) {
+        crosscheck(
+            &format!("(to-uint {})", PropValue(Value::Int(val))),
+            Err(())
+        )
+    }
+}
```

Comment:
I would not separate both tests.
The expected result is easy to compute with the input.
`prop_filter` shouldn't be used for that. It's easy to just generate a range.

Reply:
Changed for a simpler form.